infrastructure/dogfood/terraform/aws-tf-module/main.tf

```diff
@@ -460,13 +461,24 @@ module "notify_slack" {
   source  = "terraform-aws-modules/notify-slack/aws"
   version = "5.5.0"
 
-  sns_topic_name = "fleet-dogfood"
+  sns_topic_name = "fleet-dogfood-p1-alerts"
 
   slack_webhook_url = var.slack_webhook
   slack_channel     = "#help-p1"
   slack_username    = "monitoring"
 }
 
+module "notify_slack_p2" {
+  source  = "terraform-aws-modules/notify-slack/aws"
+  version = "5.5.0"
+
+  sns_topic_name = "fleet-dogfood-p2-alerts"
+
+  slack_webhook_url = var.slack_webhook
```

Comment:
This will require a separate webhook from the other as the webhook URLs are tied to channels even though we have to specify a slack_channel separately in the module.

We'll have to configure a new variable and also the github action to populate it from github secrets and the like.  This could all be handled separately after the feature is merged similarly to the monitoring module itself per my previous comment.

Reply:
I thought that seemed to good to be true 😏 .  Makes sense to separate it out as you said, since the code defaults to using the p1 channel for the new alert if a separate p2 topic arn isn't specified.